exercises/binary-search/introduction.md

```diff
@@ -0,0 +1,11 @@
+# Introduction
+
+You have stumbled upon a group of mathematicians who are also singer-songwriters.
+
+They have written songs for all their favorite numbers, and, as you can imagine, they have a lot of favorite numbers.
```

Comment:
```suggestion
They have written a song for each of their favorite numbers, and, as you can imagine, they have a lot of favorite numbers.
```

I'd prefer this worded this way (it's much clear to me), but I don't know if there's only one of each number in the tests.